extensions/scarb-doc/tests/data/hello_world_doc_groups/src/test_group_4.md

```diff
@@ -0,0 +1,8 @@
+
+# test group 4
+
+- ### [Modules](./test_group_4.md-modules.md)
```

Comment:
I think this should be
```suggestion
- ### [Modules](./test_group_4-modules.md)
```
Like in extensions/scarb-doc/tests/data/hello_world_doc_groups/src/test_group_4-modules.md

Reply:
indeed 👍👍